CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing
+
+:warning: This project is very alpha, expect bugs!
+These instructions are intended for contributors to get a productive workflow when developing the plugin.
+
+This project follows [scalameta's contribution guidelines].
+Please read them for information about how to create good bug reports and submit pull requests.
+
+## Features
+
+- [x] Lint with Scalafix on compile
+- [x] Formatting with Scalafmt
+- [x] Go to definition in project sources using Semanticdb
+- [ ] Go to definition in dependencies
+- [ ] Red squigglies as you type
+- [ ] Auto completions
+- [ ] Signature helper (show parameter list)
+- [ ] Show type at position
+- [ ] Find references
+- [ ] Show docstring
+
+
+## Project structure
+- `language-server` contains a Scala implementation of a language server
+- `vscode-extension` contains a Visual Studio Code extension, implementing a client for the language server
+- `test-workspace` directory for manually testing the plugin locally
+
+
+## Running locally
+
+First you need to have the following applications installed
+
+- `git`
+- `sbt`
+- Visual Studio Code (`code` from the console)
+- `npm`
+
+To try out the language-server locally, it's best to keep two open terminal
+windows.
```

Comment:
I think this is  not essential, because once you run `sbt language-server/publishLocal`, you can forget about it and continue with the vscode extension.

Reply:
I'm hesitant to advertise that workflow since this guide is primarily intended for local development of the plugin. This plugin requires more work to be usable for day-to-day for novice Scala users.

Reply:
I personally use two terminal windows since it avoids exiting/starting sbt between the test workspace and language-server project.

Reply:
Don't mind my comment, I realised that it makes sense when you simultaneously change the server code and test the client.